assignments/prototypes.js

```diff
@@ -23,6 +33,19 @@
   * should inherit destroy() from GameObject's prototype
 */
 
+function CharacterStats(c){
```

Comment:
@sukhadagholba This is technically accurate, but I would avoid referring to variables as 1 letter(unless you are referring to index as i), or abstract.  Its best practices to use something descriptive of what the variable is.  This will make future developers looking at your code much happier.